modules/33acrossBidAdapter.js

```diff
@@ -32,17 +31,41 @@ function _isViewabilityMeasurable(element) {
 }
 
 function _getViewability(element, topWin, { w, h } = {}) {
-  return utils.getWindowTop().document.visibilityState === 'visible'
+  return topWin.document.visibilityState === 'visible'
     ? _getPercentInView(element, topWin, { w, h })
     : 0;
 }
 
+function _mapAdUnitPathToElementId(adUnitCode) {
+  let id = null;
+
+  if (utils.isGptPubadsDefined()) {
+    const isMatchingAdSlot = utils.isSlotMatchingAdUnitCode(adUnitCode);
+    const matchingAdSlot = window.googletag.pubads().getSlots().find(isMatchingAdSlot);
+
+    if (matchingAdSlot) {
+      id = matchingAdSlot.getSlotElementId();
```

Comment:
`matchingAdSlot.getSlotElementId()` should be returned from this function, otherwise this function will always return `null`.